public/data/viktorCodes.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Victor Nkwanyuo",
+    "type": "personal",
+    "bio": " Full-Stack Software Engineer && Open Source Enthusiast!",
+    "avatar": "https://github.com/viktorCodes.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/viktorCodes",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/nviktordev",
+        "icon": "twitter"
+      }
+      ,
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/viktor-dev/",
+        "icon": "linkedin"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Freelancing",
+        "date": "August 2022",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started freelancing",
+        "url": "https://www.eddiejaoude.io/"
```

Comment:
I believe you should update this URL. Or you could remove this line no `31`. 

Because this has no meaning here unless you are saying that you started Freelancing at EddieHub

```suggestion
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix